ios/Classes/types/ARTClientOptions+AblyFlutterClientOptions.h

```diff
@@ -0,0 +1,11 @@
+#import "ARTClientOptions.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ARTClientOptions (AblyFlutterClientOptions)
```

Comment:
I can see what you're trying to do by using a category here but I am concerned that your new property is mutable when, as I imagine, for the lifetime of an instance it will not need to change once initially set.

Perhaps this might be an example where composition would serve more accurately what you're looking to achieve. Something like:

```objc
@interface AblyFluterClientOptions : NSObject

-(instancetype)initWithClientOptions:(ARTClientOptions *)clientOptions
                     hasAuthCallback:(BOOL)hasAuthCallback NS_DESIGNATED_INITIALZER;

@property (nonatomic, readonly) ARTClientOptions* clientOptions;
@property (nonatomic, readonly) BOOL hasAuthCallback;

@end
```

Reply:
Done, 167da5cf297b94e8025bb6f27ebc6abbec1075ef